services/http/api/lib/models/measurement.ts

```diff
@@ -1,18 +1,33 @@
 import { ModelRelatedNodesI, NeogmaInstance, NeogmaModel } from "neogma";
-import { UserModel, UserInstance } from "./user";
 import { DateInstance, DateModel } from "./date";
+import { HourInstance, HourModel } from "./hour";
+import { UserInstance, UserModel } from "./user";
 
-export type MeasurementProperties = {
+export type UserMeasurementProperties = {
     type: string,
```

Comment:
How come we're using this intermediate node type. What does the 'type' field represent here? I think we can merge UserMeasurement and Value, yeah? My preference is a single node called `Measurement` that has a 'value' field which is a number.

Reply:
Like this:
![Image 3](https://user-images.githubusercontent.com/4219194/122258375-d259ba00-ce85-11eb-8638-ea57f803205d.jpeg)

Reply:
Unfortunately, if you define 2 relationships with the same type of node, neogma will just overwrite the former with the latter. It's still possible to query with just the 1 relationship though (direction doesn't really matter apparently...) so I'm just going to keep `'Recorded'` and discard `'RecordedBy'`.

Reply:
I see! Yes directionality doesn't actually matter and we can query using just the Recorded label. I think the reverse relationship was an artifact from when I didn't really understand neo4j.